dts/bindings/gpio/rohm,bd8lb600fs.yaml

```diff
@@ -18,7 +20,6 @@ properties:
   ngpios:
     type: int
     required: true
-    const: 8
     description: Number of gpios supported
```

Comment:
Perhaps extend the current description to mention, that for non-daisy chained devices, the property should be set to 8?

Reply:
done